outscale-dev/cluster-api-provider-outscale
# RouteTable and Internetservice Unit test

Add unit test for RouteTable and InternetService:

* manual tilt launch
* tiltignore
* use buildkit with cache
* Integrate service internetsvc in osccluster_controller
* Use internetsvc instead of netsvc
* add internet service unit test
* Integrate routesvc
* Add route table unitest
* Replace netsvc by securitysvc
* Use assert
* Use split and splitmock
 * Gofmt/gofmt check in makefile
 * Check gofmt in github action
